main_test.go

```diff
@@ -196,8 +196,27 @@ func bincmp(ts *testscript.TestScript, neg bool, args []string) {
 		return
 	}
 	if data1 != data2 {
-		if _, err := exec.LookPath("diffoscope"); err != nil {
-			ts.Logf("diffoscope is not installing; skipping binary diff")
+		if _, err := exec.LookPath("diffoscope"); true || err != nil {
```

Comment:
What's with the 'true' here?

Reply:
A leftover from local testing :) Thanks for spotting.